src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -208,7 +209,7 @@ static void Main()
 
         var comp = (CSharpCompilation)verifier.Compilation;
         var symbol = comp.GetMember<NamedTypeSymbol>("C");
-        Assert.Equal("<>F0__C", symbol.MetadataName);
+        AssertEx.Equal("<>FE3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855__C", symbol.MetadataName);
```

Comment:
>AssertEx.Equal [](http://example.com/codeflow?start=8&length=14)

Just curious: Why use `AssertEx.Equal()` rather than `Assert.Equal()`?

Reply:
When the string is this long, only AssertEx will show the entire string when the assert fails.